src/main/resources/application.yaml

```diff
@@ -22,6 +22,16 @@ devservice:
     credentialPath: ${GOOGLE_SERVICE_CREDENTIAL_PATH}
     sheetId: ${GOOGLE_SERVICE_SHEET_ID}
     enabled: ${GOOGLE_SERVICE_ENABLED:false}
+  support-requests:
+    spreadsheetId: ${SUPPORT_REQUESTS_SPREADSHEET_ID}
+    credentialPath: ${GOOGLE_SERVICE_CREDENTIAL_PATH}
```

Comment:
If we use the existing GoogleCredentials then we don't need this.
Also we need to check whether to keep spreadsheetId in configuration or take it in the API call... Let's discuss

Reply:
Yes, we can discuss this.